### PR TITLE
[CL-77] Change device update to match spec

### PR DESCRIPTION
**Pull request by guarde**

### CODE REVIEW

File: src/controllers/cats.ts [hunk 1]
```diff
@@ -22,7 +19,16 @@ export function createCat(
...
 }
+
+// export function updateCat(
+// 	user: User,
+// 	request: Request,
+// 	response: Response,
+// 	next: NextFunction
+// ) {
+// 	// TODO
+// }
```
Comment (stevelw) [inline src/controllers/cats.ts]: Private comments shouldn't be committed.

File: src/controllers/devices.ts [hunk 2]
```diff
@@ -24,24 +24,23 @@ export function postUpdate(
...
 	const schema = {
+		id: 'string',
 		lat: 'number',
 		lon: 'number',
 		accuracy: 'number,optional',
 		sat: 'string,optional',
 		batt: 'number,optional',
 	};
-	const payload = request.body;
+	const payload = request.body.body;
```
Comment (stevelw) [inline src/controllers/devices.ts]: body.body?

File: src/endpoints.json [hunk 3]
```diff
@@ -41,8 +41,11 @@
 		"method": "PATCH",
 		"authentication": "device-uuid",
 		"body": {
-			"lat": "number",
-			"lon": "number"
+			"body": {
```
Comment (stevelw) [inline src/endpoints.json]: I think you've doubled your bodies.

File: src/controllers/cats.ts (hunk 1 above)
Comment (guarde) [reply src/controllers/cats.ts]: Oops, too many things at once

**Pull request merged**